package.json

```diff
@@ -20,10 +20,10 @@
     "release": "standard-version",
     "security": "npm audit",
     "test:only": "cross-env NODE_ENV=test jest",
-    "test:watch": "cross-env NODE_ENV=test jest --watch",
-    "test:coverage": "cross-env NODE_ENV=test jest --collectCoverageFrom=\"src/**/*.js\" --coverage",
+    "test:watch": "npm run test:only -- --watch",
+    "test:coverage": "npm run test:only -- --collectCoverageFrom=\"src/**/*.js\" --coverage",
     "pretest": "npm run lint",
-    "test": "cross-env NODE_ENV=test npm run test:coverage",
+    "test": "npm run test:coverage",
```

Comment:
Don't remove `cross-env` from scripts, it is break testing on windows and some old linux

Reply:
`test:only` uses cross-env

Reply:
oh, my mistake, all file :+1: